dev/PagerControl/InteractionTests/PagerControlTests.cs

```diff
@@ -0,0 +1,572 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System;
+using Common;
+using Windows.UI.Xaml.Tests.MUXControls.InteractionTests.Infra;
+using Windows.UI.Xaml.Tests.MUXControls.InteractionTests.Common;
+
+#if USING_TAEF
+using WEX.TestExecution;
+using WEX.TestExecution.Markup;
+using WEX.Logging.Interop;
+#else
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.VisualStudio.TestTools.UnitTesting.Logging;
+#endif
+
+using Microsoft.Windows.Apps.Test.Automation;
+using Microsoft.Windows.Apps.Test.Foundation;
+using Microsoft.Windows.Apps.Test.Foundation.Controls;
+using static Windows.UI.Xaml.Tests.MUXControls.InteractionTests.PagerControlTestPageElements;
+
+namespace Windows.UI.Xaml.Tests.MUXControls.InteractionTests
+{
+    [TestClass]
+    public class PagerControlTests : PagerControlTestBase
```

Comment:
Why are the tests and helpers split into different files? I suspect this would make navigating the test a little more annoying, especially from the web.

Reply:
Without the split up, the file was around 700ish lines long which isn't much fun to navigate. I can revert that if desired though.